app/src/main/res/values/strings.xml

```diff
@@ -13,4 +13,5 @@
     <string name="outfit_settings">Outfit\nSettings</string>
     <string name="closet">Closet</string>
     <string name="todo">TODO</string>
+    <string name="v">v</string>
```

Comment:
Why don't you have the ">" character stored in a string resource? Why just the "v" string.